wee/package.json

```diff
@@ -66,6 +66,7 @@
     "react-dom": "18.3.1",
     "react-icons": "^5.2.1",
     "react-leaflet": "^4.2.1",
+    "react-tooltip": "^5.28.0",
```

Comment:
why was a react tooltip installed? i don't see it being imported at the top of the scrape result page. We already have a nextui-org/tooltip that is being used for example in the navbar